### DISCUSSION TITLE
WordPress

### DISCUSSION THREAD

**Discussion by alphadeveloper622** [community/community#59660] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

https://github.com/alphadeveloper622/Wordpress-Plugin-UserList
Please give a star if it was helpful
I couldn't find a WordPress plugin production to display a list of users for their posts.
So I made it for people like me.

**Comment by tuves** [comment]
Hi @alphadeveloper622, we’re glad you’re here 😀

Thanks for sharing your (project/website/etc) with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
